requirements.txt

```diff
@@ -43,3 +43,6 @@ daphne==3.0.1
 GitPython~=3.1.27
 drf-spectacular==0.25.1
 django-cprofile-middleware==1.0.5
+# flake8 and django-nose are needed for sonar scan in openimis-be-invoice_py
```

Comment:
I think this comment can be less specific. All modules have similar structure and therefore all of them need flake and django-nose. However if that's not the case and both packages are required by invoice_py specifically then se could put them in setup.py in module's repository. 

```suggestion
# flake8 and django-nose are needed for sonar scan in backend modules
```